### DISCUSSION TITLE
Mouse hover effect on range of days (v8)

### DISCUSSION THREAD

**Discussion by oudoulj** [gpbl/react-day-picker#1196] [Support]
Is it possible to achieve a **mouseover/hover effect** when selecting a **date range** in v8 ?

Like on [Airbnb's react-dates](http://airbnb.io/react-dates/?path=/story/daterangepicker-drp--default)
It would be sweet to be able to toggle that feature with a prop on the [range](https://react-day-picker-next.netlify.app/basics/selecting-days#selecting-a-range-of-days) mode :smile: 
Unless it's already doable and I missed it...
Thank you !

![calendar mouseover example](https://user-images.githubusercontent.com/6011092/112840379-8a55a300-909f-11eb-848e-8f9ded4b34cc.png)

PS : I found the same request in [this very old](https://github.com/gpbl/react-day-picker/issues/202) thread

**Comment by jomifepe** [comment]
Any update on this? This feature was in v7 and I don't think there's a way to have it in v8.

**Comment by gpbl** [comment]
I think this is achievable with the current CSS. What have you tried?

**Comment by berkin** [comment]
it is possible with modifiers.

```
        const middleDaysStyle = {
	        background: "var(--rdp-background-color)"
        };


	const [middleDays, setMiddleDays] = useState<Matcher | Matcher[]>([]);


	const handleMouseEnter: DayMouseEventHandler = day => {
		if (selected?.from) {
			setMiddleDays({
				from: addDays(selected.from, 1),
				to: addDays(day, -1)
			});
		}

	<DayPicker
		...
		onDayMouseEnter={handleMouseEnter}
		modifiers={{ middleDays }}
		modifiersStyles={{
			middleDays: middleDaysStyle
		}}
/>
	

	
```
